packages/babel-jest/src/index.js

```diff
@@ -33,7 +36,7 @@ export const createTransformer = (options: any): Transformer => {
       supportsStaticESM: false,
     },
     compact: false,
-    plugins: (options && options.plugins) || [],
+    plugins: (options && options.plugins) || [babelTransformRuntimePlugin],
```

Comment:
```suggestion
    plugins: (options && options.plugins || []).concat(babelTransformRuntimePlugin),
```

but I don't think we should add it at all, honestly...